### PR TITLE
extract message format into base package

### PR DESCRIPTION
**Pull request by jvehent**

### CODE REVIEW

File: cmd/systrack/main.go [hunk 1]
```diff
@@ -3,49 +3,48 @@ package main
 import (
 	"log"
 
+	"go.mozilla.org/systrack"
```
Comment (jvehent) [inline cmd/systrack/main.go:6]: We use `go.mozilla.org` for all internal packages and app. I set it up in https://github.com/mozilla-services/gomoz/commit/a6b23a73638416045415f80acb4073ed4c1c6690

File: cmd/systrack/main.go [hunk 2]
```diff
@@ -3,49 +3,48 @@ package main
...
 func main() {
+	var (
+		p   systrack.PkgMsg
+		err error
+	)
 	hook, err := mozlogrus.NewKinesisHook("secops-dumper")
 	logger := logrus.New()
 	logger.Hooks.Add(hook)
-	logger.Formatter = &mozlogrus.MozLogFormatter{LoggerName: "secops-dumper", Type: "app.log"}
+	logger.Formatter = &mozlogrus.MozLogFormatter{LoggerName: "systrack", Type: "pkg"}
```
Comment (jvehent) [inline cmd/systrack/main.go:21]: The idea is to use `Type` to switch the parsing in `systrack-lambda` to the corresponding message type.

File: messages.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		"localip":      p.LocalIP,
+		"instanceid":   p.AwsInstanceID,
+		"instancetype": p.AwsInstanceType,
+		"instancetags": p.AwsInstanceTags,
+		"ami":          p.AwsAmiID,
+		"pkgname":      p.PkgName,
+		"pkgversion":   p.PkgVersion,
+		"pkgtype":      p.PkgType,
+		"pkgarch":      p.PkgArch,
+	}
```
Comment (jvehent) [inline messages.go:36]: I wonder if there's a way to define a custom tag in the message structs above and use them here.

Also, we probably should match these keys to the json tags.

**Comment by ameihm0912** [comment]
Closing this out of date PR off

**Pull request closed without merging**